src/views/Portal/components/ApplicationForm/index.view.tsx

```diff
@@ -93,6 +95,8 @@ const ApplicationForm: React.FC = () => {
   const { user } = useAuth0()
   const submitData = async () => {
     try {
+      setProgress(ApplicationPages.MLH) // this won't be saved
```

Comment:
I think we need to run the save function here to update the page state.

If you want to replicate the bug, submit an application. This will return you to page 0. Refresh the browser, you'll notice while it's loading the bar is not filled for a few seconds and that's because session storage has the incorrect value in the page state.

Reply:
Ah, yeah, I didn't want to save the progress here since the MLH responses are not saved. I though the progress would be set to 100% if the application is submitted though.

I will take a look and try to fix this tomorrow.

Reply:
@krishnakpandian Updated to update progress on submit.